### PR TITLE
Navigation bar

### PR DESCRIPTION
**Pull request by makoteq**

### CODE REVIEW

**Comment by makoteq** [comment]
There are so many files because I used fontello as icons.

File: navbar/styl.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+body{
```
Comment (nikitadhiman) [inline navbar/styl.css]: styl.css?

**Comment by nikitadhiman** [comment]
so does that means every time we have to use fontello, we would have so many files or is there some work around. Because we can take have so many files everytime. Why not directly use https://fontawesome.com/icons?d=gallery if possible.

**Comment by makoteq** [comment]
fixed

**Comment by nikitadhiman** [comment]
please squash your commits, then this is ready to go. :)

**Comment by makoteq** [comment]
Sorry, but how can I do this?

**Comment by nikitadhiman** [comment]
In case you want to merge all your commits into one, squash commit option can be used. 
Simple way of doing it would be (First do a git status and make sure your work directory is clean)
git reset --soft HEAD~5 (where you would like to merge last 5 commits to one)
git commit --edit -m "your-message-for-squashed-commits"

**Comment by makoteq** [comment]
I must squash all commits?

**Comment by makoteq** [comment]
I squash commits and merge them with master

**Comment by nikitadhiman** [comment]
Great job, closing this for now. While we merge the changes for navbar, this would definitely be considered.

**Pull request closed without merging**